services/web-server/src/utils/userIdFromIdentity.js

```diff
@@ -0,0 +1,14 @@
+const { decode } = require('./codec');
+
+module.exports = (identity) => {
+  let [strategy, encodedUserId] = identity.split('/');
+
+  if (strategy === 'mozilla-auth0') {
+    // Reverse the username appending, stripping the username.
+    if (encodedUserId.startsWith('github|') || encodedUserId.startsWith('oauth2|firefoxaccounts|')) {
+      encodedUserId = encodedUserId.replace(/\|[^|]*$/, '');
+    }
+  }
```

Comment:
This is correct, but a bit odd to have something specific to a strategy embedded in a utility function -- good fodder for a followup refactor?